index.html

```diff
@@ -60,6 +61,25 @@ <h2>Introduction</h2>
       Glitches and high latency are bad for the user experience, so if any of these occur it can be useful for the
       application to be able to detect this and possibly take some action to improve the playout.
     </p>
+    <p>
+
+          The {{AudioContext/AudioPlayoutStats}} under AudioContext is a dedicated object for statistics reporting;
```

Comment:
Add ref for AudioContext (use {{AudioContext}})

Also in other places

Reply:
DoneDone